### DISCUSSION TITLE
Add Tiktok to the `social.toml`

### DISCUSSION THREAD

**Discussion by AidaIssayeva** [razonyang/hugo-theme-bootstrap#188] [Ideas]
Would be cool to add Tiktok to the list of social media. 
The url will be `https://www.tiktok.com/@YOURUSERNAME`

Although not sure if Font Awesome has its logo

**Comment by razonyang** [comment]
Fixed by e0764b6acd4d7f1494c962f4f04a76ffebd171aa. But I'm not quite sure whether it works, because TikTok block my location and my VPN location, so I'm not able to test it myself.

**Reply by AidaIssayeva** [reply]
@razonyang you are faster than me 😄 ! Closing [my pr ](https://github.com/razonyang/hugo-theme-bootstrap/pull/190)

**Reply by AidaIssayeva** [reply]
tested and verified: everything works including the font awesome

**Reply by razonyang** [reply]
Np, thanks for your feedback. 😄
